.gitignore

```diff
@@ -1,6 +1,27 @@
-# Don’t commit the following directories created by pub.
-.dart_tool
-.pub/
+# Miscellaneous
```

Comment:
Please revert the change to this file. Ignore your IDE files in `~/.gitignore_global`

Reply:
corrected